README.md

```diff
@@ -394,7 +394,7 @@ or subclass Flusswerk Metrics-Bean:
 ```java
 @Component
 public class Metrics
-    extends de.digitalcollections.flusswerk.spring.boot.starter.monitoring.Metrics {
+    extends Metrics {
```

Comment:
Rekursion?

Reply:
Strange, could it be some some refactoring magic running wild? Also, maybe a different internal class name might be better. Since the default Spring bean is created in a config it can easily be different from the bean name.